ges/Example_GES/src/main.cpp

```diff
@@ -12,14 +17,18 @@
 #define miso            Ethercat::pdoTx.Struct.miso
 #define mosi            Ethercat::pdoRx.Struct.mosi
 
+// Set PDO sizes
+const int PDORX_size = 40;
```

Comment:
Why these sizes?

Reply:
Because array of 10 32-bit numbers = 320 bits with 8 bits/byte = 40 bytes.